scipy/stats/stats.py

```diff
@@ -1017,8 +1017,11 @@ def moment(a, moment=1, axis=0):
     """
     Calculates the nth moment about the mean for a sample.
 
-    Generally used to calculate coefficients of skewness and
-    kurtosis.
+    A moment is a specific quantitative measure of the shape of a set of points.
+    If the points represent probability density, then the zeroth moment is the
+    total probability (i. e. one), the first moment is the mean, the second
+    moment is the variance, the third moment is the skewness and the fourth
+    moment divided by the square of the variance is the kurtosis.
```

Comment:
I'm not sure about this paragraph. First central moment is not the mean, it's zero. Skewness is typically defined to be scale-free, so it's not just 3rd central moment. And I'm not sure there's a need to bring any of this into the docstring. 

I'd suggest to just stress that this a central moment (maybe, an example) and add an explicit formula.